Stata/plus/d/dlw_primus.ado

```diff
@@ -48,7 +54,7 @@ program define dlw_primus, rclass
 					error 1
 				}
 			} // dta type
-			else if "`=lower(`dlibType')'"=="do" { // only one file matched/subscribed - load the file  													
+			else if "`=lower("`dlibType'")'"=="do" { // only one file matched/subscribed - load the file  													
```

Comment:
This code is prompt to errors. It is better to use double compound quotes. Like the following 

``"`=lower("`dlibType'")'"'==`"dta"' ` // (there is an extra (`) in this code because of the formatting, but it should not go there).